packages/example/package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "chai": "3.5.0",
     "cross-env": "6.0.3",
-    "cypress-example-kitchensink": "https://github.com/cypress-io/cypress-example-kitchensink.git#bd93f68568bc0a412b2c4dd2ddbc3b7cb89e5799",
```

Comment:
Why was this bumped?

Reply:
We should add in summary/pr title if we need to bump.

Reply:
I committed this change to the wrong branch, doesn't need changing. Good catch, will undo.

Reply:
Had meant to put this change in a different PR.